<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 2009 Nokia Corporation and/or its subsidiary(-ies).
** Contact: Qt Software Information (qt-info@nokia.com)
**
** This file is part of the Qt Designer of the Qt Toolkit.
**
** $QT_BEGIN_LICENSE:LGPL$
** Commercial Usage
** Licensees holding valid Qt Commercial licenses may use this file in
** accordance with the Qt Commercial License Agreement provided with the
** Software or, alternatively, in accordance with the terms contained in
** a written agreement between you and Nokia.
**
** GNU Lesser General Public License Usage
** Alternatively, this file may be used under the terms of the GNU Lesser
** General Public License version 2.1 as published by the Free Software
** Foundation and appearing in the file LICENSE.LGPL included in the
** packaging of this file.  Please review the following information to
** ensure the GNU Lesser General Public License version 2.1 requirements
** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
**
** In addition, as a special exception, Nokia gives you certain
** additional rights. These rights are described in the Nokia Qt LGPL
** Exception version 1.0, included in the file LGPL_EXCEPTION.txt in this
** package.
**
** GNU General Public License Usage
** Alternatively, this file may be used under the terms of the GNU
** General Public License version 3.0 as published by the Free Software
** Foundation and appearing in the file LICENSE.GPL included in the
** packaging of this file.  Please review the following information to
** ensure the GNU General Public License version 3.0 requirements will be
** met: http://www.gnu.org/copyleft/gpl.html.
**
** If you are unsure which license is appropriate for your use, please
** contact the sales department at qt-sales@nokia.com.
** $QT_END_LICENSE$
**
*********************************************************************</comment>
 <class>qdesigner_internal::TreeWidgetEditor</class>
 <widget class="QDialog" name="qdesigner_internal::TreeWidgetEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Tree Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="itemsTab" >
      <attribute name="title" >
       <string>&amp;Items</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidget" >
            <property name="focusPolicy" >
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="toolTip" >
             <string>Tree Items</string>
            </property>
            <column>
             <property name="text" >
              <string>1</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="buttonsLayout" >
            <item>
             <widget class="QToolButton" name="newItemButton" >
              <property name="toolTip" >
               <string>New Item</string>
              </property>
              <property name="text" >
               <string>&amp;New</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="newSubItemButton" >
              <property name="toolTip" >
               <string>New Subitem</string>
              </property>
              <property name="text" >
               <string>New &amp;Subitem</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="deleteItemButton" >
              <property name="toolTip" >
               <string>Delete Item</string>
              </property>
              <property name="text" >
               <string>&amp;Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>28</width>
                <height>23</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="moveItemLeftButton" >
              <property name="toolTip" >
               <string>Move Item Left (before Parent Item)</string>
              </property>
              <property name="text" >
               <string>L</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="moveItemRightButton" >
              <property name="toolTip" >
               <string>Move Item Right (as a First Subitem of the Next Sibling Item)</string>
              </property>
              <property name="text" >
               <string>R</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="moveItemUpButton" >
              <property name="toolTip" >
               <string>Move Item Up</string>
              </property>
              <property name="text" >
               <string>U</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="moveItemDownButton" >
              <property name="toolTip" >
               <string>Move Item Down</string>
              </property>
              <property name="text" >
               <string>D</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="showPropertiesButton" >
              <property name="text" >
               <string>Properties &amp;>></string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qdesigner_internal::TreeWidgetEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>440</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel" >
     <x>373</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qdesigner_internal::TreeWidgetEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>556</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel" >
     <x>562</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
